packages/lesswrong/components/form-components/CoauthorsListEditor.tsx

```diff
@@ -98,7 +100,7 @@ const CoauthorsListEditor = ({ value, path, document, classes, label, updateCurr
           classes={classes}
         />
       </div>
-      <div className={classes.checkboxContainer}>
+      {isEAForum && <div className={classes.checkboxContainer}>
```

Comment:
I guess this is fine because the only thing this checkbox does is toggle `hasCoauthorPermission` values, which already have a default value of `true`